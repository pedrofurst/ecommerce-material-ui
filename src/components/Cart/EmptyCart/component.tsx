import { Typography } from '@material-ui/core';
import useStyles from './styles';

type EmptyCartPropsType = {};

function EmptyCart(props: EmptyCartPropsType) {
  const classes = useStyles();
  return (
    <div className={classes.emptyCartContainer}>
      <Typography className={classes.emptyCartText}>
        Your cart is empty.
      </Typography>
    </div>
  );
}

export default EmptyCart;
