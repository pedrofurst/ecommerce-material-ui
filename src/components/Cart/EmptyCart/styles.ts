import { makeStyles } from '@material-ui/core/styles';

export default makeStyles({
  emptyCartContainer: {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    height: '100%',
  },
  emptyCartText: {
    fontSize: 36,
    fontWeight: 'bold',
    textAlign: 'center',
  },
});
