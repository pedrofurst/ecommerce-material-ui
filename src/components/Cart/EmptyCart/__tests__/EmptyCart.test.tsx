import renderer from 'react-test-renderer';
import EmptyCart from '../component';

describe('<EmptyCart />', () => {
  it('Should have empty cart message', () => {
    const component = renderer.create(<EmptyCart />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
