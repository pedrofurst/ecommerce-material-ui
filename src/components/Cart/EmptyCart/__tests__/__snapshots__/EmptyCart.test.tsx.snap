// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyCart /> Should have empty cart message 1`] = `
<div
  className="makeStyles-emptyCartContainer-1"
>
  <p
    className="MuiTypography-root makeStyles-emptyCartText-2 MuiTypography-body1"
  >
    Your cart is empty.
  </p>
</div>
`;
